__tests__/app.test.js

```diff
@@ -174,3 +174,112 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: Responds with new posted comment object", () => {
+    const newComment = {
+      username: "rogersop",
+      body: "You should probably shorten this article",
+    };
+
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body).toMatchObject({
```

Comment:
It would be better for the comment to be on an appropriate key rather than just being the response body itself